Makefile

```diff
@@ -41,6 +39,7 @@ FFTW_CONFIG += --enable-sse2 --enable-fma
 endif
 
 ifeq ($(OS),Windows_NT)
+CONFIG += --host=$(ARCH)-w64-mingw32-gcc
```

Comment:
run ldd on it. that doesn't necessarily mean julia itself isn't found, it may mean one of its dependencies